packages/lib/src/core/core.ts

```diff
@@ -80,6 +80,10 @@ class Core {
 
         this.createCoreModules();
 
+        if (!this.paymentMethodsResponse?.paymentMethods?.length) {
+            this.createPaymentMethodsList();
+        }
```

Comment:
Maybe we don't need this if statement? We could directly call `this.createPaymentMethodsList`.
In the `PaymentMethodsResponse` there is a check in place for this!